### PR TITLE
fix: fsgid

### PR DESCRIPTION
**Pull request by mfshao**
Link to Jira ticket if there is one:

### Environments
qa-covid

### Description of changes
- fix mounted `/pd` unaccessable by user issue

### CODE REVIEW

File: qa-covid19.planx-pla.net/manifest.json [hunk 1]
```diff
@@ -22,7 +22,7 @@
     "tube": "707767160287.dkr.ecr.us-east-1.amazonaws.com/gen3/tube:2021.03",
     "fluentd": "fluent/fluentd-kubernetes-daemonset:v1.2-debian-cloudwatch",
     "spark": "707767160287.dkr.ecr.us-east-1.amazonaws.com/gen3/gen3-spark:2021.03",
-    "hatchery": "707767160287.dkr.ecr.us-east-1.amazonaws.com/gen3/hatchery:2021.03",
+    "hatchery": "quay.io/cdis/hatchery:fix_dockstore-ids",
```
Comment (mfshao) [inline qa-covid19.planx-pla.net/manifest.json:25]: will update once this change is merged and released

**Comment by PlanXCyborg** [comment]
# qa-covid19.planx-pla.net/manifest.json
## :warning: Services on branch
- portal
- hatchery

**Comment by PlanXCyborg** [comment]
# qa-covid19.planx-pla.net/manifest.json
## :warning: Services on branch
- portal
- hatchery

**Comment by PlanXCyborg** [comment]
# qa-covid19.planx-pla.net/manifest.json
## :warning: Services on branch
- portal

**Review by atharvar28 — APPROVED** [review]

**Review by paulineribeyre — APPROVED** [review]

**Pull request merged**